src/components/structures/MatrixChat.tsx

```diff
@@ -1559,73 +1587,73 @@ export default createReactClass({
                 !SettingsStore.getValue("feature_cross_signing"),
             );
         }
-    },
+    }
 
-    showScreen: function(screen, params) {
-        if (screen == 'register') {
+    showScreen(screen: string, params?: {[key: string]: any}) {
+        if (screen === 'register') {
```

Comment:
this is the sort of thing that should be stuck into an `enum` and `switch` for a later PR imo